.github/workflows/integration-tests.yml

```diff
@@ -101,7 +101,7 @@ jobs:
         run: composer testwp-experimental --no-interaction
 
       - name: Run integration tests (multi site)
-        if: ${{ ! matrix.experimental }}
+        if: ${{ matrix.php != 8.0 && ! matrix.experimental }}
```

Comment:
What issue are we fixing here?

Reply:
The 8.0 task would run tests with and without coverage, which is redundant.